flight/PiOS/Common/pios_port.c

```diff
@@ -0,0 +1,623 @@
+#include <pios.h>
```

Comment:
it also looks like this only applies to serial ports - it might be worth adding that to the name space to make it specific. as opposed to ADC ports, CAN ports, etc.